src/Symfony/Component/Security/Core/User/AdvancedUserInterface.php

```diff
@@ -32,6 +32,7 @@
  *
  * @see UserInterface
  * @see AccountStatusException
+ * @deprecated This interface is deprecated as of 3.4 and will be removed in 4.0.
```

Comment:
`@deprecated since 3.4, to be removed in 4.0` something like that i believe. Also missing a runtime deprecation? i.e. `trigger_error`

Reply:
`@deprecated since version 3.4, will be removed in 4.0.`
(but no runtime notice because that'd trigger it for existing classes in core)